### ISSUE TITLE
Feature request: ImpactFlow import

### ISSUE THREAD

**Issue by turoczy** [calagator/calagator#521]
Given that Portland startup [ImpactFlow](https://impactflow.com) is gaining traction with a lot of events, it would be great if we could add them to the sources supported by the import feature. I'm not sure if they have the right metadata to facilitate that. But I thought I'd ask.

**Comment by reidab** [comment]
@turoczy This seems potentially possible. Could you drop a few example public event links in here for us to look at? There didn't seem to be any linked from their site.

**Comment by turoczy** [comment]
@reidab Yeah. Those are more difficult to find than I realized. And most the events I was mentioning are now over. 😕 

Here's one. I'll try to track down some more.

https://impactflow.com/event/newtech-pdx-virtualaugmented-reality-1809

**Comment by turoczy** [comment]
@reidab I should have one more today (because I'm creating it). But I'm also happy to set up some dummy events. If that would be helpful, just let me know 😉

**Comment by turoczy** [comment]
https://impactflow.com/event/introducing-the-built-up-festival-2446 (live until May 1)
